src/protocol/page.rs

```diff
@@ -92,6 +127,14 @@ pub mod events {
 pub mod methods {
     use crate::protocol::Method;
     use serde::{Deserialize, Serialize};
+    use super::PrintToPdfOptions;
+
+
+    #[derive(Debug, Deserialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct ReturnDataObject {
```

Comment:
For consistency, could you rename this `PrintToPdfReturnObject`? When it comes time to use a build script or macros to generate these protocol structs, having a consistent naming pattern will help the transition.